src/app/curriculum-courses/chapter-details/chapter-details.page.ts

```diff
@@ -445,8 +448,11 @@ export class ChapterDetailsPage implements OnInit, OnDestroy {
     );
     if (this.childContents.length && !this.isBatchNotStarted) {
       const firstChild = this.loadFirstChildren(this.chapter);
-      this.navigateToChildrenDetailsPage(firstChild, 1);
-    } else {
+      const telemetryDetails = {
+        pageId: PageId.COURSE_DETAIL,
```

Comment:
PageId is incorrect

Reply:
Was a mistake, will change to "CHAPTER_DETAILS"